components/Layout/GuestLayout.js

```diff
@@ -0,0 +1,19 @@
+import Head from 'next/head'
+import Navbar from './Navbar'
+import "../../styles/Index.css"
```

Comment:
Try to search possibility to use absolute import path in next.js
so instead of "../../styles/Index.css" you can import with "styles/Index.css" directly.
some configuration needed for this. In CRA old versions you can add in .env NODE_PATH=src. But latest version not support anymore, need to do from other config.

Reply:
This way:
![image](https://user-images.githubusercontent.com/27492820/67632239-afa31800-f8d3-11e9-8ed5-cff1100de57e.png)

Reply:
No need another library ? Just create jsconfig.json ?